tests/integrations/realtiup/deploy.sh

```diff
@@ -15,9 +15,14 @@ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh
 $TIUP_BIN_DIR update playground
 
 cd ../../..
-# Run TiUP
-$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor \
-	--pd.binpath ./bin/pd-server --kv.binpath ./bin/tikv-server --db.binpath ./bin/tidb-server --tiflash.binpath ./bin/tiflash --tag pd_test \
-	> $CUR_PATH/playground.log 2>&1 &
+if [ ! -d "bin" ] || [ ! -e "bin/tikv-server" ]; then
```

Comment:
Shouldn't we check all needed binaries rather than just the TiKV?

Reply:
okay, I can add them

Reply:
<img width="1041" alt="image" src="https://github.com/tikv/pd/assets/53859786/dd2ee8db-5980-4f69-a715-020c8348da37">
added